src/NUnitFramework/tests/Constraints/DelayedConstraintTests.cs

```diff
@@ -241,7 +241,7 @@ public void ThatPollingCallsDelegateCorrectNumberOfTimes()
             Assert.That(PollCount, Is.EqualTo(4).After(110, 25));
         }
 
-        [Test]
+        [Test, Platform(Exclude = "MACOSX", Reason = "Doesn't seem to work correctly with timing, something to ponder later")]
```

Comment:
Drop this as a re-run of the build caused MacOs to pass. The machine github uses for MacOs seems to be very slow.

Reply:
I can see that other tests have this exclude as well, so I assume that it does happen from time time. Still, removed.

Reply:
...and it failed again :)